package.json

```diff
@@ -6,9 +6,15 @@
     "install-server": "(cd ./server && npm install)",
     "install-client": "(cd ./frontend && npm install --legacy-peer-deps)",
     "install": "npm run install-client && npm run install-server",
-    "postinstall": "(cd ./frontend && npm run build)"
+    "postinstall": "(cd ./frontend && npm run build)",
+    "heroku-postbuild": "ng build"
```

Comment:
ng build here would throw an error I think. there is no angular module in this folder. This is why I added the post install in the previous line.